content/en/docs/concepts/configuration/manage-resources-containers.md

```diff
@@ -31,11 +31,11 @@ If the node where a Pod is running has enough of a resource available, it's poss
 allowed) for a container to use more resource than its `request` for that resource specifies.
 However, a container is not allowed to use more than its resource `limit`.
 
-For example, if you set a `memory` request of 256 MiB for a container, and that container is in
+For example, if you set a memory `request` of 256 MiB for a container, and that container is in
 a Pod scheduled to a Node with 8GiB of memory and no other Pods, then the container can try to use
 more RAM.
 
-If you set a `memory` limit of 4GiB for that container, the kubelet (and
+If you set a memory `limit` of 4GiB for that container, the kubelet (and
```

Comment:
Same issue here. In this case, `memory` is the value under `limits`, so keeping the original would be good or you could use the full `spec.containers[].resources.limits.memory`.